File: FE-Regione-Abruzzo/abruzzo/src/app/model/ticket.model.ts
import { TicketStateModel } from './ticketStateModel.model';
import { CategoriaTicketModel } from './categoriaTicket.model';

export class TicketModel {
    id!: number;
    code!: string;
    state!: TicketStateModel;
    category!: CategoriaTicketModel;
    subcategory!: string;
    openingDate?: string;
    applicant!: string;
}

export interface IncidentTicketModel {
  statoIncident: string;
  codice: string;
  categoria: CategoriaTicketModel;
  sottoCategoria: string;
  dataApertura: string;
  richiedente: string;
}

File: BE-Regione-Abruzzo/src/main/java/com/accenture/ra/dto/response/TicketModel.java
package com.accenture.ra.dto.response;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TicketModel {

    private Long id;

    private String code;

    private TicketStateModel state;

    private String category;

    private String subcategory;

    private LocalDate openingDate;

    private String applicant;
}